spec/config-spec.coffee

```diff
@@ -1106,6 +1126,62 @@ describe "Config", ->
         expect(atom.config.get('foo.bar.str', scope: ['.source.js'])).toBe 'omg'
         expect(atom.config.get('foo.bar.str', scope: ['.source.coffee'])).toBe 'ok'
 
+      describe 'when a schema is added after config values have been set', ->
+        schema = null
+        beforeEach ->
+          schema =
+            type: 'object'
+            properties:
+              int:
+                type: 'integer'
+                default: 2
+              str:
+                type: 'string'
+                default: 'def'
+
+        it "the values set respect the new schema", ->
```

Comment:
This `it` reads a bit weird since the `it the..` doesn't really flow, maybe `respects the new schema when values are set` or something?